src/App.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
+import { GlobalStyle } from "@styles/GlobalStyle";
+
+const App = () => {
```

Comment:
App is default and run of the mill name. I think you should replace it with Modules or just like that.